plugins/lsp_v.lua

```diff
@@ -0,0 +1,13 @@
+-- Author: Rohan Vashisht: https://github.com/RohanVashisht1234
+
+-- mod-version:3
+
+local lspconfig = require "plugins.lsp.config"
+local common = require "core.common"
+local config = require "core.config"
+
+local installed_path = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_v"
+
+lspconfig.v_lsp.setup(common.merge({
```

Comment:
`lspconfig.v_lsp` is not a thing.

Reply:
Looks like even with this fixed, you still need a working install of `V`.

Reply:
oh! I just forgot about this pr. sorry, I'll fix this and think about it.